components/esm/loaddial.hpp

```diff
@@ -31,7 +31,7 @@ struct Dialogue
         Greeting = 2,
         Persuasion = 3,
         Journal = 4,
-        Deleted = -1
+        Unknown = -1 // Used for deleted dialogues
```

Comment:
The naming was changed to "Unknown", and then a comment added that previous naming "Deleted" was actually correct. Why?